### ISSUE TITLE
Images: image alt tags

### ISSUE THREAD

**Issue by kimpham54** [dulibrarytech/digitaldu-frontend#64]
Priority from accessibility task force:
use abstract or description field in MODS

**Comment by kimpham54** [comment]
div hero doesn't have image alt
all others implemented for thumbnails, alt tags
